pkg/volume/aws_ebs/aws_util.go

```diff
@@ -202,3 +221,66 @@ func getCloudProvider(cloudProvider cloudprovider.Interface) (*aws.Cloud, error)
 
 	return awsCloudProvider, nil
 }
+
+func findNvmeVolume(findName string) (device string, err error) {
+	devices, err := ioutil.ReadDir("/dev")
+	if err != nil {
+		return "", fmt.Errorf("error listing /dev directory: %v", err)
+	}
+	for _, device := range devices {
+		name := device.Name()
+		if !strings.HasPrefix(name, "nvme") {
+			continue
+		}
+		glog.V(6).Infof("found nvme %q", name)
+
+		// Skip partition devices
+		{
+			num := 0
+			ns := 0
+			partition := 0
+			tokens, err := fmt.Sscanf(name, "nvme%dn%dp%d", &num, &ns, &partition)
```

Comment:
i think it is always "n1" per aws doc http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/device_naming.html